packages/docs-browser/src/main.js

```diff
@@ -128,7 +128,8 @@ DocsBrowserApp.propTypes = {
   rootNodes: PropTypes.arrayOf(PropTypes.shape({
     entityName: PropTypes.string,
     key: PropTypes.string
-  }))
+  })),
+  documentDetailFormName: PropTypes.string
```

Comment:
With https://github.com/tocco/tocco-client/pull/1416 I added two input props: listLimit and listFormName. So we should use a consistent naming. Should we rename documentDetailFormName to detailFormName?

Reply:
that's what I wanted to call the prop first, but there are also detail forms for the domains and folders which we might want to be able to override too at some point